zk/datadog_checks/zk/zk.py

```diff
@@ -96,7 +96,8 @@ class ZookeeperCheck(AgentCheck):
 
     # example match:
     # "Zookeeper version: 3.4.10-39d3a4f269333c922ed3db283be479f9deacaa0f, built on 03/23/2017 10:13 GMT"
-    version_pattern = re.compile(r'(\d+\.\d+\.\d+)')
+    # This regex matches the entire version rather than <major>.<minor>.<patch>
+    metadata_version_pattern = re.compile('Zookeeper version: ([^,]+)')
```

Comment:
(note) Please make sure the version output is always prefixed with `Zookeeper version` across versions.

Reply:
It looks like this is the pattern based received from the stat command based on what's here:

https://github.com/DataDog/integrations-core/blob/2fb91c09935976c45f85cdd20213a9f8abfd2a87/zk/datadog_checks/zk/zk.py#L16-L20